scripts/kconfig/kconfigfunctions.py

```diff
@@ -0,0 +1,78 @@
+#
+# Copyright (c) 2018-2019 Linaro
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+import os
+import re
+
+# Types we support
+# 'string', 'int', 'hex', 'bool'
+conf_file = os.path.join(os.environ['PROJECT_BINARY_DIR'], "include/generated/generated_dts_board.conf")
+
+dt_defines = {}
+if os.path.isfile(conf_file):
+    for line in open(conf_file, 'r', encoding='utf-8'):
+        if re.search('=', line):
```

Comment:
Can do `if '=' in line:` instead (and get rid of the `import re`).

Reply:
Done.